modules/ROOT/pages/pricing-metrics.adoc

```diff
@@ -0,0 +1,138 @@
+| mllp | mllp-listener
+.4+| netsuite | deleted-object-listener | modified-object-listener | modified-record-listener | new-record-listener
+| pubsub | message-listener
+.7+| salesforce | deleted-object-listener | modified-object-listener | new-object-listener | replay-channel-listener | replay-topic-listener | subscribe-channel-listener | subscribe-topic-listener
+.2+| sap | document-listener | function-listener
+| servicebus | listener
+| sftp | listener
+| sockets | listener
+.2+| solace | queue-listener | topic-listener
+.2+| sqs | receive-messages | receivemessages
+.3+| stripe | citizen-on-new-charge-listener | on-new-charge-listener | on-new-event-listener
+| vm | listener
+.2+| websocket | inbound-listener | outbound-listener
+|===
+
+
+// Example Use-case [1 Mule flow]: A CSV file with customer contacts needs to be uploaded to CRM on a regularly scheduled basis
+
+// image
+
+// Using a single Mule flow with four elements, this application accepts CSV files that contain contact information, and then uploads the contacts to MS Dynamics. The Mule flow accepts a CSV file by polling a local folder at frequent intervals that are set with a Scheduler (event source) component. 
+
+// The columns of the CSV file contain selected contact information, such as first name, last name, phone number, and email. These columns are mapped to each of the respective fields in a specific Dynamics CRM account, and the rows are uploaded.
+
+=== Mule Flow Generated by APIKit and used for the routing of APIKit Requests
+
+APIkit is a tool that simplifies the implementation of APIs by automatically generating a minimal set of Mule flows based on the API specification. Each APIKit router endpoint counts as a distinct Mule flow. These Mule flows do not have an event source and are used for the handling of HTTP requests for a particular method and path of an API.
+
+==== Example of APIKit Request:
+
+Example 1.b - This flow is used for the routing of APIKit requests. In this case, the flow handles the GET request in the `/reservation` path.
+
+[source,xml]
+----
+<flow name="get:\reservation:cocheras-puerto-madero-api-config">
+        <logger level="INFO" message="#[output json --- attributes.queryParams]" />
+</flow>
+----
+
+// Example Use-case [12 flows]: New customer profile data needs to be moved from Marketing Cloud to Master Data Management (MDM) 
+
+// Mule Flow with Event Source [1]: Main Mule Flow with the HTTP listener that routes message to appropriate flow generated by APIKit based on the message content.
+
+// image 
+
+// Mule Flows used for routing APIKit Requests [11]:
+
+// image
+
+[[non-billable-flows]]
+=== Mule Flows that do not Count against your Anypoint Platform Package Allotment 
+
+Mule Flows that do not have an event source and are not used for the routing of APIKit requests are not charged against your Anypoint Platform package allotment. These are Mule flows primarily used for modularization of code, or code reuse purposes.
```

Comment:
```suggestion
Mule Flows that do not have an event source and are not used for the routing of APIKit requests are not charged against your Anypoint Platform package allotment. These are Mule flows primarily used to modularize code.
```

Reply:
Modularizing code can be for reuse purposes, so I don't think you need to call this out. If you want to call it out, it needs to be reworded.